app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -345,9 +345,14 @@ def alert_build_edit_screen
   end
 
   def alert_default_repeat_time
-    (@edit[:new][:expression][:eval_method] && @edit[:new][:expression][:eval_method] == "hourly_performance") ||
-      @edit[:new][:exp_event] == "_hourly_timer_" ?
-      1.hour.to_i : 10.minutes.to_i
+    if (@edit[:new][:expression][:eval_method] && @edit[:new][:expression][:eval_method] == "hourly_performance") ||
+       @edit[:new][:exp_event] == "_hourly_timer_"
+      1.hour.to_i
+    elsif @edit[:new][:expression][:eval_method] && @edit[:new][:expression][:eval_method] == "dwh_generic"
```

Comment:
The UI code does not send or knows about :always_evaluate, that is only on the server side